CommandModule/Loss_of_Two Fuel_Cells/checklist.json

```diff
@@ -1,8 +1,8 @@
 {
   "Spacecraft": 0,
-  "ProjectName": "EMR Loss of Two Fuel Cells",

```

Comment:
Why?

Reply:
Didn't mean for that to be on the same pull request

Reply:
Also emr was my shorthand name for emergency